### PR TITLE
[DPC-4107] Update docsV1.md

### PR DESCRIPTION
**Pull request by ashley-weaver**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-4107

## 🛠 Changes

Updated instructions for uploading a public key, creating an attestation, and deleting a group.

## ℹ️ Context for reviewers

(Background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers.)

## ✅ Acceptance validation

Uploading a public key
image

Creating an attestation
image

Deleting a group
image

## 🔒 Security Implications

- [ ] Adds a new software dependency or dependencies
- [ ] Modifies or invalidates one or more of our security controls
- [ ] Stores or transmits data that was not stored or transmitted before
- [ ] Requires additional review of its security implications

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: common/docsV1.md [hunk 1]
```diff
@@ -1094,7 +1094,7 @@ Details on Provenance Resources are given in the <a href="https://www.hl7.org/fh
 - **Timestamp:** Time when attestation was made.
 - **Reason:** Reason for the attestation (currently only: http://hl7.org/fhir/v3/ActReason#TREAT is supported).
 - **Organization ID:** The agent making the attestation referenced by their Organization Resource ID. 
-  - _Your Organization ID can be found by referencing the {id} variable in the resource object of your practitioner._
+  - _Your Organization ID can be found on the Organization page in the DPC Portal._
```
Comment (jdettmannnava) [inline common/docsV1.md:1097]: Probably want to link

File: common/docsV1.md [hunk 2]
```diff
@@ -1094,7 +1094,7 @@ Details on Provenance Resources are given in the <a href="https://www.hl7.org/fh
 - **Timestamp:** Time when attestation was made.
 - **Reason:** Reason for the attestation (currently only: http://hl7.org/fhir/v3/ActReason#TREAT is supported).
 - **Organization ID:** The agent making the attestation referenced by their Organization Resource ID. 
-  - _Your Organization ID can be found by referencing the {id} variable in the resource object of your practitioner._
+  - _Your Organization ID can be found on the <a href="https://sandbox.dpc.cms.gov/api/v1/Organization">Organization</a> page in the DPC Portal._
```
Comment (jdettmannnava) [inline common/docsV1.md:1097]: Oops, I meant to the instructions higher in the page

Comment (ashley-weaver) [reply common/docsV1.md:1097]: oops, my bad

File: common/docsV1.md [hunk 3]
```diff
@@ -1094,7 +1094,7 @@ Details on Provenance Resources are given in the <a href="https://www.hl7.org/fh
 - **Timestamp:** Time when attestation was made.
 - **Reason:** Reason for the attestation (currently only: http://hl7.org/fhir/v3/ActReason#TREAT is supported).
 - **Organization ID:** The agent making the attestation referenced by their Organization Resource ID. 
-  - _Your Organization ID can be found by referencing the {id} variable in the resource object of your practitioner._
+  - _Your Organization ID can be found in the DPC Portal by following <a href="https://dpc.cms.gov/docsV1.html#find-organization-id"> these instructions</a>._
```
Comment (jdettmannnava) [inline common/docsV1.md:1097]: This can be just href="#...

**Review by jdettmannnava — COMMENTED** [review]
LGTM; will approve after Eric Hayward has a look

**Review by jdettmannnava — APPROVED** [review]

**Pull request merged**